src/Settings/Identity/Banner.jsx

```diff
@@ -66,30 +64,7 @@ return (
     <Card>
       <Icon className="ph ph-info" />
       <Content>
-        <Text>
-          To ensure the privacy of your data, all credentials and preferences you enter on this settings page will be
-          securely stored in a decentralized identity operating system (idOS). Learn more about idOS{" "}
-          <TextLink href={idOSLearnLink} target="_blank">
-            here
-          </TextLink>
-          .
-        </Text>
-
-        <Text>
-          To use idOS we recommend you to use next wallet providers:{" "}
-          <TextLink href="https://wallet.meteorwallet.app" target="_blank">
-            Meteor Wallet,
-          </TextLink>{" "}
-          <TextLink href="https://www.herewallet.app" target="_blank">
-            HERE Wallet,
-          </TextLink>{" "}
-          <TextLink href="https://wallet.nightly.app" target="_blank">
-            Nightly Wallet,
-          </TextLink>{" "}
-          <TextLink href="https://app.mynearwallet.com" target="_blank">
-            MyNearWallet
-          </TextLink>{" "}
-        </Text>
+        <Text>To begin using and displaying identity data in your settings, completed the onboarding steps below.</Text>
```

Comment:
@shelegdmitriy I think this should be removed, since it doesn't apply to users who have already linked a profile and we have the two images that direct the user to the next steps.

Reply:
Do you mean to remove this totally or leave for users who doesn't have any verifications yet?